Event/event.js

```diff
@@ -1,17 +1,81 @@
 var eventList = [];
+var FreeEvents = [];
+var BusyEvents = [];
```

Comment:
Why are the variable names inconsistent here ?

Reply:
I also think we could use let instead of var here

Reply:
I wanted to use a global variable here as the scope of both arrays is global in this code. However, in this particular case, "let" also could work (I tested this now).